_posts/2022-08-05-hacking-for-cruise-control-ui.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: "Hacking Strimzi for Cruise Control UI"
+date: 2022-10-31
+author: kyle_liberti
+---
+
+For some time now, Strimzi has offered Kubernetes native support Cruise Control allowing users to safely configure and communicate with Cruise Control through Strimzi custom resources.
+This is great but sometimes it is a better user experience to interact with applications like Cruise Control through a UI.
```

Comment:
Maybe say `User Interface` the first time, then we can use UI.

Reply:
Subjective. Maybe we can say something like: _Rather than working with custom resources, you might prefer to interact with applications like Cruise Control through a user interface._